### PR TITLE
Clean up the code by removing the utils directory

### PR DESCRIPTION
**Pull request by r0x0d**
Removing the utils directory and moving around the stuff inside of it, like dd directory and handle-sshpw to their respective modules will help clean up the project.

### CODE REVIEW

**Comment by r0x0d** [comment]
Part of: https://issues.redhat.com/browse/INSTALLER-2865

**Review by KKoukiou — COMMENTED** [review]
The changes break RPM build seemingly. Can you take a look?

**Comment by r0x0d** [comment]
> The changes break RPM build seemingly. Can you take a look?

Yeah, was trying to fix that yesterday. Will take a better look next week.

File: scripts/handle-sshpw [hunk 1]
```diff
@@ -9,7 +9,7 @@
 # it under the terms of the GNU General Public License as published by
 # the Free Software Foundation; either version 2 of the License, or
 # (at your option) any later version.
-#
+#sbin
```
Comment (r0x0d) [inline scripts/handle-sshpw:12]:
```suggestion
#
```

File: scripts/Makefile.am [hunk 2]
```diff
@@ -16,12 +16,14 @@
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
 scriptsdir = $(libexecdir)/$(PACKAGE_NAME)
-dist_scripts_SCRIPTS = run-anaconda anaconda-pre-log-gen log-capture start-module apply-updates
+dist_scripts_SCRIPTS = run-anaconda anaconda-pre-log-gen log-capture start-module apply-updates
```
Comment (KKoukiou) [inline scripts/Makefile.am:19]: Unrelated whitespace change.

File: scripts/Makefile.am [hunk 3]
```diff
@@ -16,12 +16,14 @@
...
 scriptsdir = $(libexecdir)/$(PACKAGE_NAME)
-dist_scripts_SCRIPTS = run-anaconda anaconda-pre-log-gen log-capture start-module apply-updates
+dist_scripts_SCRIPTS = run-anaconda anaconda-pre-log-gen log-capture start-module apply-updates 
 
 dist_noinst_SCRIPTS  = makeupdates makebumpver
 
-dist_bin_SCRIPTS = anaconda-cleanup instperf anaconda-disable-nm-ibft-plugin \
-                   anaconda-nm-disable-autocons
+dist_bin_SCRIPTS     = anaconda-cleanup instperf anaconda-disable-nm-ibft-plugin \
+                       anaconda-nm-disable-autocons
```
Comment (KKoukiou) [inline scripts/Makefile.am:24]: Unrelated whitespace change.

**Review by KKoukiou — CHANGES_REQUESTED** [review]
Almost there. @jkonecny12 can you please take a look as well.

File: scripts/Makefile.am (hunk 2 above)
Comment (r0x0d) [reply scripts/Makefile.am:19]:
```suggestion
dist_scripts_SCRIPTS = run-anaconda anaconda-pre-log-gen log-capture start-module apply-updates
```

File: scripts/Makefile.am (hunk 3 above)
Comment (r0x0d) [reply scripts/Makefile.am:24]: This one is bad? It aligns correctly the equal sign with the other variables in the Makefile.am 🤔

Comment (jkonecny12) [reply scripts/Makefile.am:24]: It's not bad, the issue @KKoukiou is pointing out is that the change is side effect of the commit purposed to do something else. Feel free to add another commit or PR to fix this.

**Review by jkonecny12 — COMMENTED** [review]
Nice improvement :+1: . Thanks!

Please just don't forgot to squash the commits. When commits are squashed I'm happy to ACK this ;)

File: scripts/Makefile.am (hunk 3 above)
Comment (r0x0d) [reply scripts/Makefile.am]: Done, @KKoukiou!

**Review by jkonecny12 — APPROVED** [review]
Looks great to me now. Thanks!
